scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -122,7 +161,47 @@ Param (
     [string] $EdgeE2ETestRootCAPassword = $null,
 
     [ValidateScript({($_ -as [System.Uri]).AbsoluteUri -ne $null})]
-    [string] $ProxyUri = $null
+    [string] $ProxyUri = $null,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $LoadGenMessageFrequency = $null,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $SnitchAlertUrl = $null,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $SnitchBuildNumber = "1.1",
+
+    [ValidateNotNullOrEmpty()]
+    [string] $SnitchReportingIntervalInSecs = $null,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $SnitchStorageAccount = $null,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $SnitchStorageMasterKey = $null,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $SnitchTestDurationInSecs = $null,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $LoadGen1TransportType = "amqp",
+
+    [ValidateNotNullOrEmpty()]
+    [string] $LoadGen2TransportType = "amqp",
+
+    [ValidateNotNullOrEmpty()]
+    [string] $LoadGen3TransportType = "mqtt",
+
+    [ValidateNotNullOrEmpty()]
+    [string] $LoadGen4TransportType = "mqtt",
+
+    [ValidateSet("true", "false")]
+    [string] $AmqpSettingsEnabled,
```

Comment:
Use boolean type for AmqpSettingsEnabled and MqttSettingsEnabled parameter and set true as default value.

Reply:
I would like to leave this as a string. The input is a string, and the variable is being used as a string to replace <amqpSettings__enabled> and <mqttSettings__enabled> in the deployment json.